pinot-core/src/main/java/org/apache/pinot/core/common/DataSource.java

```diff
@@ -19,9 +19,7 @@
 package org.apache.pinot.core.common;
 
 import org.apache.pinot.core.operator.BaseOperator;
-import org.apache.pinot.core.segment.index.readers.BloomFilterReader;
-import org.apache.pinot.core.segment.index.readers.Dictionary;
-import org.apache.pinot.core.segment.index.readers.InvertedIndexReader;
+import org.apache.pinot.core.segment.index.readers.*;
```

Comment:
You may want to undo wildcard imports when formatting
https://stackoverflow.com/questions/3587071/disable-intellij-starred-package-imports.

Same with several other files in this diff.

Reply:
Aah good catch. Will fix this !